packages/medusa/src/services/__tests__/swap.ts

```diff
@@ -81,10 +195,12 @@ describe("SwapService", () => {
 
     it("fails if item is returned", async () => {
       const swapService = new SwapService({
+        ...defaultProps,
         eventBusService,
       })
       const res = () =>
-        swapService.validateReturnItems_(
+        // @ts-ignore
+        swapService.validateReturnItems(
```

Comment:
This is a bit tricky since the method is protected because it's only used internally in the service and I didn't want to delete tests that we currently have.
I would also prefer if we would test this method through the other public methods but would love to hear what other think

Reply:
That makes sense @carlos-r-l-rodrigues, and since you pointed out, it seems that the method isn't used anymore so I ended up removing the method and tests